src/app/shared/services/app.service.ts

```diff
@@ -92,8 +93,10 @@ export class NSSService {
         return this._showHideManageCitationsModal.asObservable();
     }
     // -+-+-+-+-+-+-+-+-+ add regression region modal -+-+-+-+-+-+-+-+
-    private _showHideAddRegressioRegionModal: Subject<boolean> = new Subject<boolean>();
-    public setAddRegressionRegionModal(val: any) {
+    private addregressionregion: AddRegressionRegion;
+    private _showHideAddRegressioRegionModal: Subject<AddRegressionRegion> = new Subject<AddRegressionRegion>();
+    public setAddRegressionRegionModal(val: AddRegressionRegion) {
+        this.addregressionregion = val;
```

Comment:
I don't know why this is here, but I'm guessing this can be deleted, as well as line 66 where this variable is created.